crypto/dh/dh_key.c

```diff
@@ -26,6 +26,11 @@
 # define MIN_STRENGTH 80
 #endif
 
+#if defined(OPENSSL_SYS_LINUX) && !defined(FIPS_MODULE) && defined (__s390x__)
```

Comment:
To avoid repeating the same condition everywhere this should be a new macro (S390X_MOD_EXP) that is defined somewhere - perhaps in include/crypto/bn.h

Reply:
Done.